s3_file_field/views.py

```diff
@@ -17,6 +18,8 @@ class UploadInitializationRequestSerializer(serializers.Serializer):
     file_name = serializers.CharField(trim_whitespace=False)
     file_size = serializers.IntegerField(min_value=1)
     # part_size = serializers.IntegerField(min_value=1)
+    content_type = serializers.CharField(required=False, trim_whitespace=False)
+    content_disposition = serializers.CharField(required=False, trim_whitespace=False)
```

Comment:
I don't think we should take a `content_disposition ` directly from the client. It should always use stored as `attachment` (and only because we have to make a decision). The client is already providing a `file_name`, so we can just use that.

It's a separate matter for clients to choose to override `Content-Disposition` on an individual download basis, using pre-signed GET request URLs.